### PR TITLE
USHIFT-4061: change imagePullPolicy from Always to IfNotPresent

### PR DESCRIPTION
**Pull request by eslutsky**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: assets/optional/multus/07-daemonset-dhcp.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
       containers:
       - name: dhcp-daemon
         image: containernetworking-plugins-microshift
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```
Comment (ggiguash) [inline assets/optional/multus/07-daemonset-dhcp.yaml:43]: Do we need to update the rebase script as well?

Comment (eslutsky) [reply assets/optional/multus/07-daemonset-dhcp.yaml:43]: AFAIK we dont  rebase multus from upstream,
this is the reference:
https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/assets.yaml#L257

@pmtk , correct me if am wrong..

**Pull request merged**